src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManagerInterface.php

```diff
@@ -97,7 +97,7 @@ public function getCount();
      *
      * @return Collection
      */
-    public function save(array $data, int $userId, bool $breadcrumb = false);
+    public function save(array $data, ?int $userId, bool $breadcrumb = false);
```

Comment:
Could be a little BC Break but think that someone uses the CollectionManagerInterface alone would never happen and the CollectionManager class the $userId is `?int`